pdc/apps/package/models.py

```diff
@@ -27,6 +27,7 @@ class RPM(models.Model):
     release             = models.CharField(max_length=200, db_index=True)
     arch                = models.CharField(max_length=200, db_index=True)  # nosrc
     srpm_name           = models.CharField(max_length=200, db_index=True)  # package (name of srpm)
+    koji_build_target   = models.CharField(max_length=200, null=True, blank=True, db_index=True)
```

Comment:
Actually, if you want to track "RPM was built in a release" relation,
why don't you add a RPM.built_for_release = ForeignKey(Release, null=True, blank=True) field?

That's something completely unrelated to infrastructure and it's up to an external script / user to provide the correct data.